src/datepicker/styled-components.js

```diff
@@ -89,28 +76,56 @@ export const StyledCalendarHeader = styled(
 
 export const StyledMonthHeader = styled('div', (props: SharedStylePropsT) => {
   return {
+    color: props.$theme.colors.white,
+    backgroundColor: props.$theme.colors.primary,
     whiteSpace: 'no-wrap',
   };
 });
 
-function getArrowBtnStyle({$theme}: SharedStylePropsT) {
+export const StyledMonthYearSelectButton = styled('button', props => {
+  return {
+    ...props.$theme.typography.font400,
+    alignItems: 'center',
+    backgroundColor: 'transparent',
+    borderWidth: 0,
+    color: props.$theme.colors.mono100,
+    cursor: 'pointer',
+    display: 'flex',
+    ':focus': {backgroundColor: props.$theme.colors.primary500},
+  };
+});
+
+export const StyledMonthYearSelectIconContainer = styled('span', props => {
+  return {
+    alignItems: 'center',
+    display: 'flex',
+    marginLeft: props.$theme.sizing.scale500,
+  };
+});
+
+function getArrowBtnStyle({$theme, $disabled}: SharedStylePropsT) {
```

Comment:
I believe the same way we use the acronym `Btn` for `Button`, we could use for other terms that occur very often. For example, `OverriddenStatefulPopover` could be written as `OvdStfPopover`, `StyledMonthYearSelectIconContainer` as `SldMonthYearSltIcCtr`. Or maybe assign special and unique characters to represent words like `styled` `overridden` and `stateful`. This is something to be done in the far future, but I guess it could reduce some variable names considerably.